clients/client/src/main/java/org/projectnessie/client/http/HttpTreeClient.java

```diff
@@ -52,17 +55,30 @@ public ReferencesResponse getAllReferences(ReferencesParams params) {
         .queryParam("pageToken", params.pageToken())
         .queryParam("fetch", FetchOption.getFetchOptionName(params.fetchOption()))
         .queryParam("filter", params.filter())
+        .queryParam(
+            "includeExpired",
+            params.includeExpired() != null ? params.includeExpired().toString() : null)
+        .queryParam(
+            "includeTransactions",
+            params.includeTransactions() != null ? params.includeTransactions().toString() : null)
         .get()
         .readEntity(ReferencesResponse.class);
   }
 
   @Override
-  public Reference createReference(String sourceRefName, @NotNull Reference reference)
+  public Reference createReference(
+      String sourceRefName,
+      @NotNull Reference reference,
+      @Nullable Instant expireAt,
+      @Nullable Boolean txn)
```

Comment:
transaction instead of txn? same in other places (particularly user and openapi facing methods